src/capital-project/capital-project.controller.ts

```diff
@@ -33,6 +36,15 @@ import { ZodTransformPipe } from "src/pipes/zod-transform-pipe";
 export class CapitalProjectController {
   constructor(private readonly capitalProjectService: CapitalProjectService) {}
 
+  @UsePipes(new ZodTransformPipe(findCapitalProjectsQueryParamsSchema))
```

Comment:
I think we're double parsing the query parameters. Once on this line and again on line 42. 

Either approach works but we should only keep one.